app/src/main/java/com/fynov/equaleyes/appestimator/data/models/Feature.java

```diff
@@ -0,0 +1,66 @@
+package com.fynov.equaleyes.appestimator.data.models;
+
+public class Feature {
+    private String id;
+    private String name;
+    private String description;
+    private Integer price;
+    private Integer time;
+    private Boolean selected = false;
+
+    public Feature(String id, String name, String description, Integer price, Integer time) {
+        this.id = id;
+        this.name = name;
+        this.description = description;
+        this.price = price;
+        this.time = time;
+    }
+
+    public Boolean getSelected() {
```

Comment:
Pri booleanih se ponavadi uporabi `is` namesto `get`. Tak, da `isSelected` bi bilo boljše poimenovanje

Reply:
Fixed